docs/sources/guides/whats-new-in-v4-1.md

```diff
@@ -13,7 +13,7 @@ weight = 3
 ## What's new in Grafana v4.1
 - **Graph**: Support for shared tooltip on all graphs as you hover over one graph. [#1578](https://github.com/grafana/grafana/pull/1578), [#6274](https://github.com/grafana/grafana/pull/6274)
 - **Victorops**: Add VictorOps notification integration [#6411](https://github.com/grafana/grafana/issues/6411), thx [@ichekrygin](https://github.com/ichekrygin)
-- **Opsgenie**: Add OpsGenie notification integratiion [#6687](https://github.com/grafana/grafana/issues/6687), thx [@kylemcc](https://github.com/kylemcc)
```

Comment:
I'm not sure if we should change the historical changelogs?

Reply:
No. Unless there are obvious things like typos or broken links, do not change the old release notes.

Reply:
@daniellee said the opposite... https://github.com/grafana/grafana/pull/24385#pullrequestreview-407442843

Reply:
Maybe I was to fast to merge. For me, if it is fixing typos then I don't see why not?

Reply:
Sorry, I misread it. Yes, this is fine. (Brain is mush, apologies.)

Reply:
@daniellee I think at least that the PR should be topical, and I don't consider release notes to be regular documentation.

When it comes to changing release notes I think we have to be careful, and not modify changelog entries since they refer to GitHub PRs.